zstd-safe/zstd-sys/build.rs

```diff
@@ -138,7 +138,11 @@ fn compile_zstd() {
 
     config.define("ZSTD_LIB_DEPRECATED", Some("0"));
 
-    config.flag_if_supported("-flto=thin");
+    if !config.get_compiler().is_like_gnu() {
+        // gcc has a -flto but not -flto=thin
+        // Apparently this is causing crashes on windows-gnu?
```

Comment:
Maybe we can enable fat LTO for windows?

Reply:
The PR doesn't even fix windows build, so I'll need to find something else :(